clang/lib/Sema/SemaOverload.cpp

```diff
@@ -4404,14 +4404,19 @@ CompareImplicitConversionSequences(Sema &S, SourceLocation Loc,
     Result = CompareStandardConversionSequences(S, Loc,
                                                 ICS1.Standard, ICS2.Standard);
   else if (ICS1.isUserDefined()) {
+    const FunctionDecl *ConvFunc1 = ICS1.UserDefined.ConversionFunction;
```

Comment:
So I think this code should have a comment considering how complex the issue this is fixing is.

Reply:
Done